chalk-parse/src/ast.rs

```diff
@@ -230,6 +234,12 @@ pub enum ScalarType {
     Float(FloatTy),
 }
 
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
+pub enum Mutability {
```

Comment:
This is fine :) Other types are duplicated between `chalk`ir` and `chalk-parse` (like the scalars above)